model/metric.go

```diff
@@ -87,6 +89,22 @@ func (m Metric) FastFingerprint() Fingerprint {
 	return LabelSet(m).FastFingerprint()
 }
 
+func (m Metric) MarshalEasyJSON(w *jwriter.Writer) {
+	w.RawByte('{')
+	first := true
+	for k, v := range m {
+		if !first {
+			w.RawByte(',')
+		} else {
+			first = false
```

Comment:
nit: be consistent how you handle the if-else here and in the LabelSet marshaling method (I guess I prefer the if-else variant, since it only assigns the `first` variable once - not that I think it'd make a real difference).

Reply:
Good catch, I switched to the else, since it seems better :) (tests don't show much of a difference, but it looks better at least)